README.md

```diff
@@ -30,9 +26,11 @@ This portfolio site must have the following HTML files:
 
 - `index.html` must include your name somewhere.
 
-- `about.html` should include some information about you, your interests, background or similar. Only post what you are comfortable sharing.
+- `facts.html` should include a series of fun facts or more detailed information about your topic.
```

Comment:
Line 25 above is unchanged so I can't leave a comment there, but the sentence starts "This portfolio site must..."; would we want that to say "fansite" rather than "portfolio"?